LR_Tourist/TouristWebApp/Areas/Identity/Data/TouristWebAppContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace TouristWebApp.Data
+{
+    public class TouristWebAppContext : IdentityDbContext<IdentityUser>
```

Comment:
You can reuse already existing DbContext.

Reply:
fixed